buildSrc/src/main/groovy/maru.kotlin-common-minimal-conventions.gradle

```diff
@@ -11,6 +11,7 @@ java {
 }
 
 repositories {
+  mavenLocal()
```

Comment:
I feel like it's safer to comment it out and enable it only for the local debugging. I don't think it should go into the repo